releasenotes/README.md

```diff
@@ -26,30 +26,40 @@ Additional documentation:
 * [Salesforce CLI Setup Guide](https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_intro.htm)
 
 
-## 2.32.8 (March 13, 2024) [stable-rc]
+## 2.33.3 (March 20, 2024) [stable-rc]
 
 These changes are in the Salesforce CLI release candidate. We plan to include these changes in next week's official release. This list isn't final and is subject to change.
 
 ------------
 
-* FIX: Salesforce DX projects now support these [metadata types](https://github.com/forcedotcom/source-deploy-retrieve/blob/main/src/registry/metadataRegistry.json):
+* FIX: The `org user display` command no longer displays the Profile Name twice. (GitHub issue [#2762](https://github.com/forcedotcom/cli/issues/2762), plugin-user PR [#898](https://github.com/salesforcecli/plugin-user/pull/898))
 
-  * MlModelArtifact
-  * MlModelConnection
-  * MlModelSchema
+    Thank you [dwbuttler](https://github.com/dwbuttler) for noticing the problem, reporting the issue, and then simply fixing it yourself.  We love your initiative!  And we look forward to more contributions from you and our fabulous community.
 
-## 2.31.10 (March 8, 2024) [stable]
+* FIX: We've improved the error message returned when `project deploy validate` encounters a validation error with one or more metadata components. (GitHub issue [#2757](https://github.com/forcedotcom/cli/issues/2757), plugin-deploy-retrieve PR [#938](https://github.com/salesforcecli/plugin-deploy-retrieve/pull/938))
+
+* FIX: You can now successfully authenticate an org in which the associated connected app has PKCE ([Proof Key for Code Exchange](https://help.salesforce.com/s/articleView?id=sf.remoteaccess_pkce.htm&type=5)) enabled.  (sfdx-core PR [#1035](https://github.com/forcedotcom/sfdx-core/pull/1035))
```

Comment:
There are several ways you can enforce PKCE.  On the connected app is one of them.  The CLI supports PKCE however it's enforced.